### PR TITLE
Fixed activity duration based on the activity name

### PR DESCRIPTION
**Pull request by nikolayilyin**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3394)

### CODE REVIEW

**Comment by nikolayilyin** [comment]
This feature allows for configured activities to have always the same fixed duration

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
looks like a flaky test..
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

File: src/test/scala/beam/agentsim/agents/ActivitiesDurationSpec.scala [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    val activities: Seq[Activity] = events
+      .filter(e => "actstart".equals(e.getEventType) || "actend".equals(e.getEventType))
+      .map {
+        case act if act.getEventType == "actstart" =>
+          new ActStart(act.getTime, act.getAttributes.get("actType"), act.getAttributes.get("person"))
+        case act if act.getEventType == "actend" =>
+          new ActEnd(act.getTime, act.getAttributes.get("actType"), act.getAttributes.get("person"))
+      }
+
+    val activitesToDurations = mutable.HashMap.empty[String, Set[Double]]
```
Comment (dimaopen) [inline src/test/scala/beam/agentsim/agents/ActivitiesDurationSpec.scala]: a typo: activities

**Review by dimaopen — APPROVED** [review]
a small note only

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Pull request merged**